keyboards/splitkb/zima/zima.c

```diff
@@ -0,0 +1,112 @@
+/* Copyright 2019 Thomas Baart
```

Comment:
Changes in behaviour should be put in keymaps/default to contain special behaviour inside of the keymap definition.
Especially OLED behaviour should not be defaulted in the base definition, as it interferes strongly with your own definitions. I suggest limiting the base definitions to the absolute minimum, so you never have to touch the root zima folder.

Reply:
That's why it's defined weakly.  the `__attribute__((weak))` prefix means that you can replace the function in other code. 

But the reason for having it in the keyboard's c file is so that it will work for the Configurator. Eg, so that the encoder and OLED will actually do stuff when compiling from the configurator.

Reply:
Configurator support is the main goal, and I'd say doing it the way Drashna has done it is elegant.